### PR TITLE
[JENKINS-33628] Separate ItemsContent from AboutJenkins

### PR DESCRIPTION
**Pull request by fbelzunc**
This PR intends to fix [JENKINS-33628](https://issues.jenkins-ci.org/browse/JENKINS-33628).

In an enterprise environment with thousands of jobs/builds, there is a big performance degradation produced by the computation of counting jobs/builds. When Jenkins is inside a Load Balancer, usually the HTTP session finished before the computation is done and this finish with the support-bundle being corrupted.

Support bundle is critical for Jenkins support organizations.

CC @escoem @aheritier

### CODE REVIEW

**Comment by varyvol** [comment]
@fbelzunc if I am understanding this correctly, you want to move out the items data recollection so that it can be disabled on large instances and thus avoid the performance issues?

File: src/main/java/com/cloudbees/jenkins/support/impl/ItemsContent.java [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+                    out.println("======================");
+                    out.println();
+                    out.println("  * Number of jobs: " + jobTotal.n());
+                    out.println("  * Number of builds per job: " + jobTotal);
+                }
+            });
+        }
+    }
+
+    private static class Stats {
```
Comment (varyvol) [inline src/main/java/com/cloudbees/jenkins/support/impl/ItemsContent.java]: I'm not a big fan of duplicating this class. Do you think it makes sense to move it to an external class to be then used anywhere necessary?

Comment (fbelzunc) [reply src/main/java/com/cloudbees/jenkins/support/impl/ItemsContent.java]: Yes, I will submit the changes

Comment (fbelzunc) [reply src/main/java/com/cloudbees/jenkins/support/impl/ItemsContent.java]: Sorry, the class is not used in `AboutJenkins` anymore, so I just removed it from there. No need to create a new class only for this.

**Comment by fbelzunc** [comment]
@varyvol Would you mind to re-review, please? I think it should be ready to merge now :-)

**Comment by fbelzunc** [comment]
@varyvol Answering also your previous comment, you are right, the idea is to separate this from the AboutJenkins to avoid the performance issue, which usually produced also corrupted support-bundle when Jenkins is behind a load balancer - as the load balancer timeout is usually reached out.

**Pull request merged**